geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/internal/PostgresTableMetaDataManagerIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.connectors.jdbc.internal;
+
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.ClassRule;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.test.junit.categories.IntegrationTest;
+import org.apache.geode.test.junit.rules.DatabaseConnectionRule;
+import org.apache.geode.test.junit.rules.PostgresConnectionRule;
+
+@Category(IntegrationTest.class)
+public class PostgresTableMetaDataManagerIntegrationTest
+    extends TableMetaDataManagerIntegrationTest {
+
+  private static final URL COMPOSE_RESOURCE_PATH =
+      PostgresTableMetaDataManagerIntegrationTest.class.getResource("postgres.yml");
+
+  @ClassRule
+  public static DatabaseConnectionRule dbRule = new PostgresConnectionRule.Builder()
+      .file(COMPOSE_RESOURCE_PATH.getPath()).serviceName("db").port(5432).database(DB_NAME).build();
```

Comment:
Just double checking: these hardcoded port numbers are okay because of docker?

Reply:
yes, in the docker compose yml file, we specify the port in the container to open, which is then mapped to a random, open port externally. Docker (and the library we are using) handle the mapping between the ports from there.